server/data/submittedApplicationClient.ts

```diff
@@ -29,6 +29,9 @@ export default class SubmittedApplicationClient {
     })) as SubmittedApplication
   }
 
+  /**
```

Comment:
I think if these functions get deleted in this PR we don't need to add the comments in this commit?

Reply:
No changes are required to `submittedApplicationClient.ts` so these comments no longer exist.